### PR TITLE
Feature/79 device page

### LINKED ISSUE #77: Routine Page
After selecting a routine card from the list of routines, the user should be redirected to a routine page. This should list all configurations for that routine (device & time offset) and display the base alarm time.

### LINKED ISSUE #78: Devices Page

### LINKED ISSUE #79: Device Page

### LINKED ISSUE #91: Consolidate Styling
The following features should be styled consistently across the UI:
* The headers for each view.
* The alignment for headers/content in each view.
* The styling of fetch requests.

### PR DESCRIPTION
**Pull request by LanceWells**
# Issues Resolved

This PR resolves the following issues:
* #79
* #78
* #91 
* #77 

# Screenshots
| View Name | Screenshot |
|-----------|------------|
| Update Device | update_device_D |
| Add Device To Routine | add_device_to_routine_D |
| Single Routine | single_routine_D |
| Devices | devices |
| Routines | routines_D (2) |
| Account Details | accountDetails_D |

### CODE REVIEW

**Review by LindsayMichelleAllen — APPROVED** [review]

**Review by LindsayMichelleAllen — CHANGES_REQUESTED** [review]
ignore this

File: api/api.go [hunk 1]
```diff
@@ -114,6 +114,9 @@ func main() {
 			if userResponse.Error != nil {
```
Comment (LindsayMichelleAllen) [inline api/api.go:114]: - [x] Merge #108 

Looks great! Thanks for adding the screenshots. 
I think we should merge #108 first so I can deploy the changes on AWS and make sure the Public DNS doesn't change.

**Review by LindsayMichelleAllen — APPROVED** [review]

File: api/api.go (hunk 1 above)
Comment (LanceWells) [reply api/api.go:114]: What a fantastic idea! I'll get right on that!

Comment (LanceWells) [reply api/api.go:114]: That PR has been closed, and the branch is now merged!!!

**Pull request merged**